### ISSUE TITLE
[BUG] <title>deepl translation error

### ISSUE THREAD

**Issue by lumenbloom** [SillyTavern/SillyTavern#3224]
### Environment

🪟 Windows

### System

win11，edge

### Version

1.12.9

### Describe the problem

deepl does not work as expected after adding the api key .
If insiders need to test it, I can provide the api key.
【error:forbidden】

### Additional info

<img width="226" alt="123" src="https://github.com/user-attachments/assets/da7d4916-6a36-40d9-8606-2af4b9028bc8" />

### Please tick the boxes

- [X] I have explained the issue clearly, and I included all relevant info
- [X] I have checked that this [issue hasn't already been raised](https://github.com/SillyTavern/SillyTavern/issues?q=is%3Aissue)
- [X] I have checked the [docs](https://docs.sillytavern.app/) ![important](https://img.shields.io/badge/Important!-F6094E)

**Comment by Cohee1207** [comment]
I cannot reproduce this. I suspect that you're getting a geographical restriction, as 403 response code is sometimes used to indicate this. Please see the command line logs (a line that starts with "DeepL error:  ") or verify with a VPN that masks a location.

**Comment by lumenbloom** [comment]
> 我无法复制这个。我怀疑您受到了地理限制，因为 403 响应代码有时被用来表示这一点。请查看命令行日志（以 “DeepL error： ” 开头的行）或使用掩盖位置的 VPN 进行验证。
I checked my logs and found an error in the translation.
DeepL error: Forbidden {“message”: “Wrong endpoint. Use https://api.deepl.com”}
But when I checked the link, I found that it is already 404, and now the link in deepl is not this one, is it related to this?

**Comment by Cohee1207** [comment]
Thanks, this should now be fixed in staging by adding an endpoint selection. You should choose "Pro".

**Comment by lumenbloom** [comment]
> 谢谢，通过添加端点选择，这个问题现在应该可以在暂存中解决了。您应该选择 "专业版"。

Yes, my deep account is pro. If you need to test this feature, I can provide api key for free.

**Comment by Cohee1207** [comment]
You can `git switch staging` to test it yourself.
